test/integration/targets/cloudwatchlogs/playbooks/full_test.yml

```diff
@@ -0,0 +1,6 @@
+- hosts: localhost
```

Comment:
I don't see you doing anything 'clever' with this.

If you move
test/integration/targets/cloudwatchlogs/playbooks/roles/cloudwatchlogs_log_group_metric_filter/tasks/main.yml
to
test/integration/targets/cloudwatchlogs/tasks/main.yml
then you should be able to remove playbooks/full_test.yml  and runme.sh